buildlib/az-helpers.sh

```diff
@@ -54,6 +54,14 @@ function azure_log_warning() {
     echo "##vso[task.logissue type=warning]${msg}"
 }
 
+# Complete the task as "succeeeded with issues"
+function azure_complete_with_issues() {
+    test "x$RUNNING_IN_AZURE" = "xno" && return
+    msg=$1
+    set +x
+    echo "##vso[task.complete result=SucceededWithIssues;]DONE${msg}"
```

Comment:
can you pls push bad C file to this PR as temp example of how "SucceededWithIssues" looks?